.github/workflows/test.yaml

```diff
@@ -102,8 +102,7 @@ jobs:
       # version of go.
       PATH: /snap/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:${{ github.workspace }}/bin
       GOROOT: ""
-      GITHUB_PROJECT_URL: ${{ github.server_url }}/${{ github.repository }}
-      GITHUB_PULL_REQUEST: ${{ github.event.number }}
+      GITHUB_PULL_REQUEST_TITLE: ${{ github.event.pull_request.title }}
```

Comment:
run-checks still references GITHUB_PULL_REQUEST. The condition of running PR title check needs to be updated.

Reply:
I added a commit to fix this: https://github.com/snapcore/snapd/pull/14021/commits/d6a29bfada85f0148da817f114c8d3880cb4f8fb